### PR TITLE
feat(regin,host,host-deployer): qga deploy and fsdriver

### PR DESCRIPTION
**Pull request by wanyaoqi**
- host-deployer add deploy qga support
- host add qga partition fsdriver
- qga set network use fsdriver
- hostpinger sync qga status

**What this PR does / why we need it**:

**Does this PR need to be backport to the previous release branch?**:
release/3.11

### CODE REVIEW

File: pkg/hostman/guestfs/fsdriver/linux_init.go [hunk 1]
```diff
@@ -88,3 +88,26 @@ WantedBy=multi-user.target
 
 	return nil
 }
+
+func (d *sLinuxRootFs) InstallQemuGuestAgentSystemd() error {
+	var serviceName = "qemu-guest-agent.service"
```
Comment (zexi) [inline pkg/hostman/guestfs/fsdriver/linux_init.go:93]: 有没有考虑虚拟机里面本来就有 qga 的情况？或者用户自己安装 qga 会不会出现冲突？

Comment (wanyaoqi) [reply pkg/hostman/guestfs/fsdriver/linux_init.go:93]: image

@zexi 这个在别的地方做了校验

**Pull request merged**